Microsoft.Toolkit.Uwp.Services/Services/Twitter/TwitterOAuthRequest.cs

```diff
@@ -58,17 +59,17 @@ internal class TwitterOAuthRequest
         /// <returns>awaitable task</returns>
         public async Task ExecuteGetStreamAsync(Uri requestUri, TwitterOAuthTokens tokens, TwitterStreamCallbacks.RawJsonCallback callback)
         {
-            using (var request = new HttpHelperRequest(requestUri, HttpMethod.Get))
+            using (var request = new HttpRequestMessage(HttpMethod.Get, requestUri))
             {
                 var requestBuilder = new TwitterOAuthRequestBuilder(requestUri, tokens);
 
-                request.Headers.Authorization = HttpCredentialsHeaderValue.Parse(requestBuilder.AuthorizationHeader);
+                request.Headers.Authorization = AuthenticationHeaderValue.Parse(requestBuilder.AuthorizationHeader);
 
-                using (var response = await HttpHelper.Instance.GetInputStreamAsync(request).ConfigureAwait(false))
+                using (var response = await client.SendAsync(request).ConfigureAwait(false))
```

Comment:
I am not so sure. Once a request is complete and a response is received you are then reading the response as stream as opposed to requesting the response itself as stream.

They should be identical if GET is only set to return response with headers.

I could be very wrong but I did try to unify them at least once.